### ISSUE TITLE
BUG: `RangeEditor` is lacking dirty-state indication

### ISSUE THREAD

**Issue by grebaldi** [neos/neos-ui#3499]
### Description

The `RangeEditor` has no indication for when its value has changed (as opposed to every other editor).

### Steps to Reproduce

1. Configure a node type with a property that can be edited via `RangeEditor`, e.g.:
```yaml
'Vendor.Site:MyContentNode':
  # ...

  properties:
    rangeEditorExample:
      type: integer
      ui:
        label: 'RangeEditor Example'
        reloadIfChanged: true
        help:
          message: >
            This is a Range editor example.
        inspector:
          group: examples
          editor: 'Neos.Neos/Inspector/Editors/RangeEditor'
          editorOptions:
            minLabel: Invisible
            maxLabel: Opaque
            min: 0
            max: 100
            step: 5
            unit: px
```
2. Create/Edit such a node in the UI
3. Change the value via the range slider
4. Notice no orange border/ouline to indicate that the value has changed

#### Expected behavior

When the value of a `RangeEditor` is change, an orange border or outline should appear to indicate that the value has changed.

*Example: The `TextFieldEditor`*

https://github.com/neos/neos-ui/assets/2522299/128ee1a1-0407-4c84-bb94-51969f901f64

#### Actual behavior

https://github.com/neos/neos-ui/assets/2522299/9c2b0a6c-1bb8-4b7f-b041-99498080661a

### Affected Versions

Neos: >= 7.3

UI: >= 7.3
